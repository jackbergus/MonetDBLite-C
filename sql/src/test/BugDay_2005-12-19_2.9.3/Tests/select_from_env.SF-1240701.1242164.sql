select * from env where name = ( select 'prefix' from env );
select * from env where 1 in ( select 1 from env );
