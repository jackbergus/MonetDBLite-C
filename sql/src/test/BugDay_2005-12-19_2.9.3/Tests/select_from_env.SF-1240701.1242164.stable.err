stderr of test 'select_from_env.SF-1240701.1242164` in directory 'src/test/BugDay_2005-12-19_2.9.3` itself:


# 13:22:15 >  
# 13:22:15 >  Mtimeout -timeout 180 Mserver "--config=/ufs/sjoerd/Monet-virgin/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/sjoerd/src/MonetDB/devel/sql-virgin/Linux/lib/MonetDB:/ufs/sjoerd/Monet-virgin/lib/MonetDB" --set "gdk_dbfarm=/ufs/sjoerd/Monet-virgin/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/sjoerd/Monet-virgin/var/MonetDB/log" --set mapi_port=30620 --set sql_port=47090 --set xquery_port=50410 --set monet_prompt= --trace "--dbname=mTests_src_test_BugDay_2005-12-19_2.9.3" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 13:22:15 >  

# builtin opt 	config = ${prefix}/etc/MonetDB.conf
# builtin opt 	gdk_arch = 64bitx86_64-redhat-linux-gnu
# builtin opt 	gdk_version = 4.9.3
# builtin opt 	monet_pid = 19742
# builtin opt 	prefix = /ufs/sjoerd/Monet-virgin
# builtin opt 	exec_prefix = ${prefix}
# builtin opt 	gdk_dbname = tst
# builtin opt 	gdk_dbfarm = ${prefix}/var/MonetDB
# builtin opt 	gdk_debug = 8
# builtin opt 	gdk_mem_bigsize = 137438953472
# builtin opt 	gdk_vm_minsize = 137438953472
# builtin opt 	gdk_alloc_map = yes
# builtin opt 	monet_admin = adm
# builtin opt 	monet_prompt = >
# builtin opt 	monet_welcome = yes
# builtin opt 	monet_mod_path = ${exec_prefix}/lib/MonetDB:${exec_prefix}/lib/bin
# builtin opt 	monet_daemon = yes
# builtin opt 	host = localhost
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_noheaders = no
# builtin opt 	mapi_debug = 0
# builtin opt 	mapi_clients = 2
# builtin opt 	sql_port = 45123
# builtin opt 	sql_debug = 0
# builtin opt 	sql_user = monetdb
# builtin opt 	sql_schema = sys
# builtin opt 	sql_logdir = ${prefix}/var/MonetDB/log
# config opt 	prefix = /ufs/sjoerd/Monet-virgin
# config opt 	exec_prefix = ${prefix}
# config opt 	gdk_dbfarm = ${prefix}/var/MonetDB/dbfarm
# config opt 	gdk_dbname = demo
# config opt 	gdk_alloc_map = no
# config opt 	gdk_embedded = no
# config opt 	gdk_debug = 0
# config opt 	monet_admin = adm
# config opt 	monet_prompt = MonetDB>
# config opt 	monet_welcome = yes
# config opt 	monet_mod_path = ${exec_prefix}/lib/MonetDB:${exec_prefix}/lib/bin
# config opt 	monet_daemon = no
# config opt 	mapi_port = 50000
# config opt 	mapi_debug = 0
# config opt 	sql_port = 45123
# config opt 	sql_debug = 0
# config opt 	sql_logdir = ${prefix}/var/MonetDB/log
# config opt 	xquery_port = 45789
# config opt 	pf_httpd_port = 8080
# config opt 	xquery_output = dm
# config opt 	xquery_cacheMB = 100
# cmdline opt 	config = /ufs/sjoerd/Monet-virgin/etc/MonetDB.conf
# cmdline opt 	gdk_debug = 10
# cmdline opt 	monet_mod_path = /ufs/sjoerd/src/MonetDB/devel/sql-virgin/Linux/lib/MonetDB:/ufs/sjoerd/Monet-virgin/lib/MonetDB
# cmdline opt 	gdk_dbfarm = /ufs/sjoerd/Monet-virgin/var/MonetDB/dbfarm
# cmdline opt 	sql_logdir = /ufs/sjoerd/Monet-virgin/var/MonetDB/log
# cmdline opt 	mapi_port = 30620
# cmdline opt 	sql_port = 47090
# cmdline opt 	xquery_port = 50410
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbname = mTests_src_test_BugDay_2005-12-19_2.9.3

# 13:22:15 >  
# 13:22:15 >  Mtimeout -timeout 60 MapiClient -lsql -u monetdb -P monetdb --host=localhost --port=47090  < select_from_env.SF-1240701.1242164.sql
# 13:22:15 >  

MAPI  = monetdb@localhost:39170
QUERY = select * from env() as env where name = ( select 'prefix' from env() as env );
ERROR = !ERROR: zero_or_one: cardinality violation (43>1)
        !ERROR: zero_or_one: operation failed.

# 13:22:15 >  
# 13:22:15 >  Done.
# 13:22:15 >  

