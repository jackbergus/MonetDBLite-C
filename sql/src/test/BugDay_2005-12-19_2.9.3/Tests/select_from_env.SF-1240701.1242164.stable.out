stdout of test 'select_from_env.SF-1240701.1242164` in directory 'src/test/BugDay_2005-12-19_2.9.3` itself:


# 19:10:37 >  
# 19:10:37 >   mserver5 "--config=/ufs/manegold/_/scratch0/Monet/Testing/Current/prefix./etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/Testing/Current/prefix./lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/Testing/Current/prefix./lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/Testing/Current/prefix./lib64/MonetDB5/bin" --set "gdk_dbfarm=/ufs/manegold/_/scratch0/Monet/Testing/Current/prefix./var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/manegold/_/scratch0/Monet/Testing/Current/prefix./var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=33236 --set xrpc_port=46350 --set monet_prompt= --trace  "--dbname=mTests_src_test_BugDay_2005-12-19_2.9.3" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 19:10:37 >  

# MonetDB server v5.7.0, based on kernel v1.25.0
# Serving database 'mTests_src_test_BugDay_2005-12-19_2.9.3', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://rig.ins.cwi.nl:33236/
# MonetDB/SQL module v2.25.0 loaded

Ready.
#function user.main():void;
#    clients.quit();
#end main;

Over..

# 11:51:03 >  
# 11:51:03 >  Mtimeout -timeout 60 MapiClient -lsql -umonetdb -Pmonetdb --host=localhost --port=35781  < select_from_env.SF-1240701.1242164.sql
# 11:51:03 >  

% .env # table_name
% name # name
% varchar # type
% 18 # length
[ "gdk_arch"	]
[ "gdk_version"	]
[ "prefix"	]
[ "exec_prefix"	]
[ "gdk_dbname"	]
[ "gdk_dbfarm"	]
[ "gdk_debug"	]
[ "gdk_alloc_map"	]
[ "gdk_vmtrim"	]
[ "monet_admin"	]
[ "monet_prompt"	]
[ "monet_welcome"	]
[ "monet_mod_path"	]
[ "monet_daemon"	]
[ "host"	]
[ "mapi_port"	]
[ "mapi_noheaders"	]
[ "mapi_debug"	]
[ "mapi_clients"	]
[ "sql_debug"	]
[ "standoff_ns"	]
[ "standoff_start"	]
[ "standoff_end"	]
[ "config"	]
[ "gdk_nr_threads"	]
[ "mapi_open"	]
[ "xrpc_open"	]
[ "xrpc_port"	]
[ "mal_listing"	]
[ "gdk_embedded"	]
[ "mero_msglog"	]
[ "mero_errlog"	]
[ "mero_pidfile"	]
[ "mero_controlport"	]
[ "mal_init"	]
[ "mapi_autosense"	]
[ "sql_optimizer"	]
[ "default_pipe"	]
[ "replication_pipe"	]
[ "accumulator_pipe"	]
[ "recycler_pipe"	]
[ "cracker_pipe"	]
[ "sidcrack_pipe"	]
[ "datacell_pipe"	]
[ "octopus_pipe"	]
[ "mitosis_pipe"	]
[ "datacyclotron_pipe"	]
[ "derive_pipe"	]
[ "dictionary_pipe"	]
[ "gdk_mem_pagebits"	]
[ "gdk_mem_bigsize"	]
[ "monet_pid"	]
[ "monet_version"	]

# 13:22:15 >  
# 13:22:15 >  Done.
# 13:22:15 >  

